marklogic-data-hub-central/ui/src/components/entities/mapping/mapping-card.test.tsx

```diff
@@ -55,10 +55,18 @@ describe("Mapping Card component", () => {
       getByRole = renderResults.getByRole;
       queryAllByRole = renderResults.queryAllByRole;
       getByText = renderResults.getByText;
+      getByTestId = renderResults.getByTestId;
     });
 
     expect(getByRole("edit-mapping")).toBeInTheDocument();
```

Comment:
We should remove the expect statement to verify getByRole("edit-mapping") is in the document or not. The next mouseover statement does the needful here and will anyway fail if it's not available.